homeassistant/components/google_assistant/smart_home.py

```diff
@@ -95,6 +95,28 @@
     return response
 
 
+@HANDLERS.register('action.devices.IDENTIFY')
+async def async_devices_identify(hass, data, payload):
+    """Handle action.devices.IDENTIFY request.
+    
+    https://developers.google.com/actions/smarthome/develop/local
+    """
+    device = payload.get('device')
```

Comment:
This is not correct. 

The only thing that Google is detecting is Home Assistant instances. And so we can't nor should identify entities. We should instead identify Home Assistant as the hub, and set it to be `isLocalOnly` to true.